Swabbr.Infrastructure/Abstractions/DatabaseContextBase.cs

```diff
@@ -39,23 +40,51 @@ public virtual async ValueTask<DatabaseContext> CreateNewDatabaseContext(string
         ///     sql command text.
         /// </summary>
         /// <remarks>
-        ///     Call this last in the sql query definition process.
+        ///     If the <paramref name="navigation"/> has no sorting
+        ///     order specified or if <paramref name="sortingColumn"/>
+        ///     is null or empty, no sorting is appended to the query.
+        ///     Note that <paramref name="sortingColumn"/> is optional, 
+        ///     which means that sorting will be ignored if none is
+        ///     specified.
         /// </remarks>
         /// <param name="cmdText">SQL query.</param>
         /// <param name="navigation">Navigation control.</param>
-        protected static void ConstructNavigation(ref string cmdText, Navigation navigation)
+        /// <param name="sortingColumn">The column to sort by.</param>
+        protected static void ConstructNavigation(ref string cmdText, Navigation navigation, string sortingColumn = null)
```

Comment:
Remove the `ref` This now creates a reference to an object which is already referenced by C#. This increases the refcount by 1 and _could_ potentially delay the GC